hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/AbstractScheduledExecutorContainerHolder.java

```diff
@@ -32,14 +32,16 @@
         implements ScheduledExecutorContainerHolder {
 
     final NodeEngine nodeEngine;
+    final TaskLifecycleHook taskLifecycleHook;
```

Comment:
This field can be avoided if the `ScheduledExecutorMemberBin` initialises the constructor function in the constructor, like `ScheduledExecutorPartition` does (and you get a nice lambda + remove `nodeEngine` field)